<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartAlarmDialog</class>
 <widget class="QDialog" name="SmartAlarmDialog">
  <property name="windowTitle">
   <string>Smart Alarm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblWakeTime">
       <property name="text">
        <string>Target Wake Time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="timeTargetWake"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblDesiredHours">
       <property name="text">
        <string>Desired Sleep (hrs):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spinDesiredHours">
       <property name="minimum">
        <double>0</double>
       </property>
       <property name="maximum">
        <double>12</double>
       </property>
       <property name="value">
        <double>8</double>
       </property>
       <property name="singleStep">
        <double>0.5</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblRecommendedBedtimeLabel">
       <property name="text">
        <string>Recommended Bedtime:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblOptimalTime">
       <property name="text">
        <string>--:--</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnCalculate">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoxOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoxCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
